app/components/vertical_ellipsis_menu_component/vertical_ellipsis_menu_component.html.haml

```diff
@@ -0,0 +1,4 @@
+.vertical-ellipsis-menu{ "data-controller": "vertical-ellipsis-menu" }
+  %i.fa.fa-ellipsis-v{ "data-action": "click->vertical-ellipsis-menu#toggle" }
+  .vertical-ellipsis-menu-content{ "data-vertical-ellipsis-menu-target": "content" }
+    = content
```

Comment:
I think that a partial could have also done a `yield` here instead of the `= content`. Then we wouldn't even need a component.